### ISSUE TITLE
Dashboard Card Help Text Disappears Off Screen

### ISSUE THREAD

**Issue by mdavis1982** [laravel/nova-issues#2549]
- Laravel Version: 7.7.1
- Nova Version: 3.4.0
- PHP Version: 7.4.3
- Database Driver & Version: N/A
- Operating System and Version: macOS 10.15.3
- Browser type and version: Chrome 81.0.4044.122

### Description:
When the last card on a dashboard has some help text, it disappears off screen:

![image](https://user-images.githubusercontent.com/199807/80280707-a7e6f380-86fd-11ea-8c72-92f7d63ec446.png)

### Steps To Reproduce:
Add some help text to the last card in a row.

**Comment by preliot** [comment]
Same issue over here. Also can't read rest of help text with vertical (brower) scrollbar.

**Comment by preliot** [comment]
Still an issue dear stale bot..

**Comment by sanderbaas** [comment]
I use this ugly css workaround untill this issue is resolved.

```
[id^=popover] {
  margin-left: -250px;
}
```

Custom css can be loaded by adding a Nova style in the NovaServiceProvider boot method:

```
    public function boot()
    {
        parent::boot();

        Nova::style('custom-theme', 'css/custom-nova.css');
    }
```

**Comment by preliot** [comment]
@sanderbaas Copied your code in my already existing custom theme, works! Thanks!

**Comment by crynobone** [comment]
Tried multiple configuration changes but nothing seem to work. Some finding regarding the package:

* It hasn't been updated for quite sometimes
  - last release was 10 months ago
  - last commit 6 months ago
* It still using Popper.js v1 and no sign it will be updated to v2.
* Issue reported on the repository https://github.com/Akryum/v-tooltip/issues/581

Using above CSS may work for this case by it might be risky to have it as default in the core as it might not work with placement other than `top` and `bottom`.

**Comment by mdavis1982** [comment]
@crynobone In a project where I had this issue with Popper JS (reported here: https://github.com/popperjs/popper-core/issues/457), I ended up using the HubSpot Tether library (http://tether.io) which doesn't have this issue.

**Comment by crynobone** [comment]
@mdavis1982 thank you for the info. Will look into this soon.

**Comment by jonnott** [comment]
Still not fixed as of 3.21 :(

**Comment by crynobone** [comment]
PR to change placement from `top` to `top-start` has been accepted.

![CleanShot 2021-02-05 at 08 32 00](https://user-images.githubusercontent.com/172966/106978164-aa02d600-6796-11eb-8dcd-a2858c4ec0d2.png)

**Comment by crynobone** [comment]
Fixed and released on v3.21.1
